### ISSUE TITLE
Running PDK native packages on Windows under ConEmu fails

### ISSUE THREAD

**Issue by garethr** [puppetlabs/pdk#220]
Output from running `pdk`

```
PS > Get-Module -Name PuppetDevelopmentKit

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     0.4.0      PuppetDevelopmentKit                pdk


PS C:\Users\garet> pdk
ConEmu blocks ANSICON injection
ConEmu blocks ANSICON injection
ConEmu blocks ANSICON injection
Program 'ansicon.exe' failed to run: The system cannot find the file specifiedAt C:\Program
Files\WindowsPowerShell\Modules\PuppetDevelopmentKit\PuppetDevelopmentKit.psm1:8 char:3
+   &$env:DEVKIT_BASEDIR\private\tools\bin\ansicon.exe $env:RUBY_DIR\bi ...
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~.
At C:\Program Files\WindowsPowerShell\Modules\PuppetDevelopmentKit\PuppetDevelopmentKit.psm1:8 char:3
+   &$env:DEVKIT_BASEDIR\private\tools\bin\ansicon.exe $env:RUBY_DIR\bi ...
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (:) [], ApplicationFailedException
    + FullyQualifiedErrorId : NativeCommandFailed
```

More context: https://github.com/cmderdev/cmder/issues/84

**Comment by DavidS** [comment]
This is already tracked at https://tickets.puppetlabs.com/browse/PDK-356

**Comment by DavidS** [comment]
This has been resolved in the 0.6 packages. ansicon now is not used under conemu.
